allofplos/article_class.py

```diff
@@ -943,11 +930,17 @@ def remote_tree(self):
     @property
     def journal(self):
         """Journal that an article was published in.
-
         Can be PLOS Biology, Medicine, Neglected Tropical Diseases, Pathogens,
         Genetics, Computational Biology, ONE, or the now defunct Clinical Trials.
+        Relies on a simple doi_to_journal transform when possible, and uses `Journal().get_plos_journal()`
+        for the "annotation" DOIs that don't have that journal information in the DOI.
         """
-        return self.get_plos_journal()
+        if 'annotation' not in self.doi:
+            journal = Journal.doi_to_journal(self.doi)
+        else:
+            journal_meta = self.root.xpath('/article/front/journal-meta')[0]
+            journal = Journal(self.doi, journal_meta).get_plos_journal()
```

Comment:
I think I'd prefer it be called `parse_plos_journal` to indicate that you're actually parsing as opposed to relying on the quick transform.